src/utils/rest-api.js

```diff
@@ -323,6 +323,27 @@ export function renameStudy(studyName, userId, newStudyName) {
     }).then((response) => response.json());
 }
 
+export function changeStudyAccessRights(studyName, userId, toPrivate) {
+    console.info('Change access rights of study ' + studyName);
+    const changeStudyAccessRightsUrl =
+        PREFIX_STUDY_QUERIES +
```

Comment:
Thre is a new getStudyUrl() to share this little piece of code. Could you use it?